packages/grant-explorer/src/features/discovery/RoundCardStat.tsx

```diff
@@ -1,18 +1,20 @@
 import { ChainId } from "common";
+import { formatUnits } from "viem";
 import { CHAINS } from "../api/utils";
 import { Badge } from "../common/styles";
 
 type RoundCardStatProps = {
   chainId: ChainId;
   matchAmount: string;
   token: string;
+  decimals: number;
```

Comment:
It's not immediately clear what this refers to. `tokenDecimals`?

Reply:
Fixed